### DISCUSSION TITLE
Homage to the Square: Exercise 2

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#3] [Show and tell]
1. Open the file `exercises/02-homage-to-the-square/exercise-2.Rmd`
2. Find an inspiration image and extract colors, or choose a palette from one of the sources above. Make sure you have at least 4 colors!
3. Update the function `make_my_homage()` to take your palette as the default value of the `colors` argument
4. Run the function `make_my_homage()` with a few different seeds and pick your output
5. Share the inspiration and output in this GitHub discussion

**Comment by Meghansaha** [comment]
I'm sorry I got excited and jumped ahead 🫣

Inspiration:
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/348464a4-b257-47e2-88bf-321a96a21966)

Output:
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/4c936ae0-2bb8-4b85-a5f0-2b6dc8099d41)

**Reply by Meghansaha** [reply]
More Inspiration: 
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/45c89206-5382-4e6e-b635-60c4744d28af)

Another Output:
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/405f98fe-bbf1-4ce9-9eb1-96fac163dc8d)

**Reply by Meghansaha** [reply]
Inspo:
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/eff9cb7d-b65a-4db3-a635-816783b6b55b)


Output:
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/6123e406-be78-4bf2-b494-372cc8d61ee9)

**Comment by thomaspara** [comment]
My inspo:
![squirrl](https://github.com/posit-conf-2023/creative-coding/assets/10082327/f466faf9-193b-4794-98e6-a785b99778e8)
My pallet: "#3d701f", "#49291e", "#dcc59d", "#6c8160", "#9b6844"
My output:
![file_show](https://github.com/posit-conf-2023/creative-coding/assets/10082327/ad5b4e63-f29d-422c-a681-3c7059130f77)

**Comment by sshaughnessy** [comment]
![seed-27](https://github.com/posit-conf-2023/creative-coding/assets/36434962/7049339f-e285-4082-a437-31d38e6e35af)

palette from coolors

**Comment by MHanken** [comment]
![Marlene Palette](https://github.com/posit-conf-2023/creative-coding/assets/49319823/2ac735ed-4da7-4a75-8fcb-d565d975b827)

![new_homage](https://github.com/posit-conf-2023/creative-coding/assets/49319823/67052633-14bf-4605-95c0-db58b61a19d6)

**Comment by Kim-s-h** [comment]
I like Wes Anderson's movie.

![ex1_2](https://github.com/posit-conf-2023/creative-coding/assets/22897443/1176fcf4-fb13-4486-b327-c6bf43335021)

**Comment by maxrogow** [comment]
heatmiser
![heatmiser_homage_square](https://github.com/posit-conf-2023/creative-coding/assets/145362959/20a21c31-b9ad-404e-9d09-9f5c3bd0858d)

**Comment by bshk93** [comment]
![ex2-homage-201](https://github.com/posit-conf-2023/creative-coding/assets/65432465/a1436db5-7efa-45da-896d-824887287eda)

Inspired by playing around on coolers

**Comment by iamciera** [comment]
<img width="297" alt="Screen Shot 2023-09-18 at 11 29 12 AM" src="https://github.com/posit-conf-2023/creative-coding/assets/1732390/9205b8d0-7cbb-45e3-bcdf-5b15d87b8478">
<img width="319" alt="Screen Shot 2023-09-18 at 11 31 43 AM" src="https://github.com/posit-conf-2023/creative-coding/assets/1732390/9c836133-4192-434b-a351-79fa134d1b25">

**Comment by datatoolbox** [comment]
![](https://user-images.githubusercontent.com/5366795/268711990-67bf5ea0-a129-4b55-91f1-21e5d6772d6f.png)

**Comment by shaunmwilkinson** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362966/11171b23-3109-423b-ad39-58ca3609e354)

**Comment by dougtommet** [comment]
![ex2-homage-7](https://github.com/posit-conf-2023/creative-coding/assets/7944225/37ac0727-1e76-4c39-bae6-ec4131a7518d)

**Comment by lwdozal** [comment]
Trying to do metallic ... still working
![metallic_blue_deep](https://github.com/posit-conf-2023/creative-coding/assets/19821171/173967ac-84b1-4daa-9a5d-9bbe33fadc2e)

got the blue!! 
![ex2-homage-47](https://github.com/posit-conf-2023/creative-coding/assets/19821171/2e5bfdc7-a85f-4662-9f05-ad2a9c783f1b)

![metallic_pink_deep](https://github.com/posit-conf-2023/creative-coding/assets/19821171/48b26b53-afeb-4099-92e0-54643b1dfe68)
![metallic_pink_light](https://github.com/posit-conf-2023/creative-coding/assets/19821171/41664145-5283-4112-a060-246bc3348f59)

**Comment by natalieoshea** [comment]
![colors_trends_2023](https://github.com/posit-conf-2023/creative-coding/assets/10425075/e1268bae-31ca-40f2-a5f3-7acbedd025c4)

Love these colors!

**Reply by natalieoshea** [reply]
![ex2-homage-555](https://github.com/posit-conf-2023/creative-coding/assets/10425075/977d1ea0-3078-46a4-8b2a-211ccefd0eaa)


Seed: 555
